care/utils/assetintegration/base.py

```diff
@@ -48,16 +66,18 @@ def _validate_response(self, response: requests.Response):
                 {"error": "Invalid Response"}, response.status_code
             ) from e
 
-    def api_post(self, url, data=None):
+    def api_post(self, url, data=None, timeout=None):
+        if timeout is None:
+            timeout = self.timeout
+        timeout = min(timeout, 60)
```

Comment:
```suggestion
    def api_post(self, url, data=None, timeout=self.timeout):
```
hardlimiting it to 60 doesn't make a difference, the request might get timed out under 30s it is better to leave it to the dev to manage the request in case they want a longer connection period for async tasks

Reply:
Vignesh suggested it to keep a max timeout limit of 60 in an EOD